checkout.html

```diff
@@ -0,0 +1,141 @@
+
+
+<!DOCTYPE html>
+<html lang="zxx">
+  <head>
+    <meta charset="utf-8">
+    <title>Adopt A Dog!</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+  </head>
+  <body>
+
+      <header>
+        <img src="images/adoptadog-logo.svg" alt="Adopt a Dog logo">
+        <h1>Seattle's premier dog adoption agency</h1>
+      </header>
+
+      <nav>
+        <a href="index.html">Home&nbsp;&nbsp;/</a>
+        <a href="dogs.html">&nbsp;&nbsp;Dogs&nbsp;&nbsp;/</a>
+        <a href="blog.html">&nbsp;&nbsp;Blog</a>
+        <a href="checkout.html"><img src="images/cart-icon.svg" alt="Cart icon"></a><button>Checkout</button>
+      </nav>
+
+          <h2>Checkout</h2>
+
+          <form method="GET">
+                <label>Name
+                    <input type="text" name="contactName">
+                </label>
+  
+                <label>Email Address
+                    <input type="email" name="contactEmail">
+                </label>
+  
+                <label>Street Address
+                    <input type="text" name="contactLocation">
+                </label>
+
+                <label>City
+                        <input type="text" name="contactLocation">
+                    </label>
+
+                <label>State
```

Comment:
Can you make this City, State, and Zip[lines 43 -100] display as one single line on your checkout.html?